src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -34,6 +34,11 @@ import { SessionError } from "../../services/helpers";
 import { useQuota } from "../../hooks/useQuota/useQuota";
 import { AuthStoreContext } from "../../context/authStore";
 import { useTranslate } from "../../hooks/useTranslate/useTranslate";
+import {
+  PaymentQRDeformedError,
+  PaymentQRMissingInfoError,
+  PaymentQRUnsupportedError,
+} from "../../utils/paymentQrValidation";
```

Comment:
can we import these errors from the `payment-qr-parser` lib instead?

Reply:
Yes, we can, have switched to using the errors from the library instead in [6b0627a](https://github.com/rationally-app/mobile-application/pull/537/commits/6b0627af0dfecf2d23917e128146da46b770dc95)!